### PR TITLE
Task-42425: Allow searching redactional spaces for super admin, super spaces manager and space manager

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**
**Current behavior:** The spaces suggestor will not display:
- Redactional spaces (with at least one redactor) on which I'm manager or when I'm a super spaces manager or super admin

**New behavior:** The spaces suggestor will not display:
- Redactional spaces (with at least one redactor) on which I'm manager or when I'm a super spaces manager or super admin

### CODE REVIEW

File: webapp/portlet/src/main/webapp/common/js/SuggesterService.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ function searchSpaces(filter, items, excludeNonRedactor, noRedactorSpace) {
     .then(resp => resp && resp.ok && resp.json())
     .then(data => {
       data.spaces.forEach((item) => {
-        if ((!noRedactorSpace || !item.redactorsCount) && (!excludeNonRedactor || item.isRedactor || !item.redactorsCount)) {
+      // canEdit = true if manager or super manager
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/common/js/SuggesterService.js]: to be removed

File: webapp/portlet/src/main/webapp/common/js/SuggesterService.js [hunk 2]
```diff
@@ -24,7 +24,8 @@ function searchSpaces(filter, items, excludeNonRedactor, noRedactorSpace) {
...
+        if ((!noRedactorSpace || !item.redactorsCount) && (!excludeNonRedactor || item.isRedactor || !item.redactorsCount) || (!excludeNonManager || item.canEdit)) {
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/common/js/SuggesterService.js:27]: Why adding item.canEdit here ?

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: webapp/portlet/src/main/webapp/common/js/SuggesterService.js (hunk 2 above)
Comment (hassenelkamel-exo) [reply webapp/portlet/src/main/webapp/common/js/SuggesterService.js:27]: it return true if user is manager or super manager to allow suggest space for them

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**